mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -24,6 +24,21 @@ trait OverrideCompletions { this: MetalsGlobal =>
       val detail: String
   ) extends ScopeMember(sym, NoType, true, EmptyTree)
 
+  class OverrideAllMember(
```

Comment:
Can we rename it to `ImplementAllMember`? I feel like there's a subtle difference between all in "OverrideAll" and "ImplementAll"

- "all" in `OverrideAll` means all the members we can override.
- "all" in `ImplementAll` means all the **abstract** members we can implement (override).

Reply:
Good idea! Renamed